Source/WebCore/page/ChromeClient.h

```diff
@@ -668,13 +668,13 @@ class ChromeClient {
 
     virtual void textReplacementSessionUpdateStateForReplacementWithID(const UnifiedTextReplacement::SessionID&, UnifiedTextReplacement::ReplacementState, const UnifiedTextReplacement::ReplacementID&) { }
 
-    virtual void removeTextIndicatorStyleForID(const UnifiedTextReplacement::SessionID&) { }
+    virtual void removeTextAnimationForID(const UnifiedTextReplacement::SessionID&) { }
 
-    virtual void cleanUpTextStylesForSessionID(const UnifiedTextReplacement::SessionID&) { }
+    virtual void cleanUpTextAnimationsForSessionID(const UnifiedTextReplacement::SessionID&) { }
```

Comment:
I think "clean up" is a bit vague; what does this actually do?

Reply:
It removed all animations associated with this id as well as all visibility markers. It's more of a final... cleanup... than something that should be strictly necessary

Reply:
is cleanUpTextAnimationsForSessionID ever called without calling removeTextAnimationForID before / after? I'm wodndering if they can just be merged into one method